src/Console/Command/HelpCommand.php

```diff
@@ -51,15 +51,15 @@ public static function toString($value): string
     /**
      * Returns the allowed values of the given option that can be converted to a string.
      *
-     * @return list<null|AllowedValueSubset|scalar>
```

Comment:
1. fix the docblock to claim that we may return null and not an array
2. not sure how we came with enumerated list of possible types here. values are proxied to Sf's OptionsResolver, which accept [mixed](https://github.com/symfony/options-resolver/blob/6.1/OptionsResolver.php#L565)

Reply:
1. 👍🏻 
2. After checking existing fixers I decided to type hint `FixerOption::$allowedValues` as `list<(callable(mixed): bool)|null|scalar>`, then adjusted this method to return only simple types (no arrays or objects, except `AllowedValueSubset` which is a special case), which doesn't change the current behavior, but would exclude callables like invocable classes or `[object, method]`. If it's wrong, sorry for letting that slip to the `master`.

Reply:
1. 👍🏻 
2. After checking existing fixers I decided to type hint `FixerOption::$allowedValues` as `list<(callable(mixed): bool)|null|scalar>`, then adjusted this method to return only simple types (no arrays or objects, except `AllowedValueSubset` which is a special case), which doesn't change the current behavior, but would exclude callables like invocable classes or `[object, method]`. If it's wrong, sorry for letting that slip to the `master`.

Reply:
ad2 - this method is taking a parameter that is a value that we proxy as-is to OptionsResolver and rely on it to validate whatever needed. They accept `mixed`, if we want to narrow-down, I think we should start validating the input. If we don't, we gonna diverge with what we pass vs what we document.

Reply:
> this method is taking a parameter that is a value that we proxy as-is to OptionsResolver and rely on it to validate whatever needed

Yes, but the value we're talking about is the validation schema, not the data to be validated.

> if we want to narrow-down, I think we should start validating the input

We do, `FixerOptionBuilder::setAllowedValues()` accepts only `list<(callable(mixed): bool)|null|scalar>`.